src/Core/Content/Media/MediaService.php

```diff
@@ -121,7 +121,7 @@ public function fetchFile(Request $request, ?string $tempFile = null): MediaFile
         }
 
         $contentType = $request->headers->get('content_type');
-        if ($contentType === 'application/json') {
+        if (strpos( $contentType , 'application/json' ) === 0) {
```

Comment:
as we require `symfony/polyfill-php8`, we can use [str_starts_with](https://www.php.net/manual/de/function.str-starts-with.php)